akka-actor-testkit-typed/src/main/scala/akka/actor/testkit/typed/TestKitSettings.scala

```diff
@@ -38,13 +40,22 @@ object TestKitSettings {
    */
   def create(config: Config): TestKitSettings =
     new TestKitSettings(config)
+
+  private object Ext extends ExtensionId[Ext] {
```

Comment:
Not an awesome name, but I guess it will never be seen anywhere since internal/private anyway?

Reply:
exactly, private

the reason is that LoggingEventFilter loads teh TestKitSettings from `intercept` so avoiding the extra work with the extension